### PR TITLE
range info sections

### PR DESCRIPTION
**Pull request by pebiantara**

### CODE REVIEW

File: src/docs/compounds/image-with-text.hbs [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        <picture class="d-block imagewtext--large__picture">
+          <source srcset="https://via.placeholder.com/470x470" media="(min-width: 992px)"/>
+          <img src="https://via.placeholder.com/350x350" class="w-100"/>
+        </picture>
+      </div>
+    </figure>
+  </div>
+  <div class="my-5">
+    <div class="container py-5"></div>
+  </div>
```
Comment (foria) [inline src/docs/compounds/image-with-text.hbs:47]: why do we need this empty DIV? We should avoid it

File: src/docs/compounds/image-with-text.hbs [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    <figure class="imagewtext imagewtext--reverse row flex-row-reverse mb-lg-2 mt-5 justify-content-center align-items-center mx-0">
+      <div class="col-12 col-lg-6 order-first order-lg-last p-0">
+        <picture class="position-absolute dots-decoration">
+          <img src="../images/pores-pink-dotes.svg">
+        </picture>
+        <picture class="position-relative">
+          <source srcset="https://via.placeholder.com/640x560" media="(min-width: 992px)"/>
+          <img src="https://via.placeholder.com/375x375" class="w-100"/>
+        </picture>
+      </div><figcaption class="col-12 col-lg-6 p-0">
```
Comment (foria) [inline src/docs/compounds/image-with-text.hbs:58]: check syntax

**Review by foria — CHANGES_REQUESTED** [review]

File: src/docs/compounds/image-with-text.hbs (hunk 1 above)
Comment (pebiantara) [reply src/docs/compounds/image-with-text.hbs:47]: oh that is for divider, let me update

File: src/docs/compounds/image-with-text.hbs (hunk 2 above)
Comment (pebiantara) [reply src/docs/compounds/image-with-text.hbs]: updated

**Review by foria — APPROVED** [review]
just replace the PNGs with SVGs

**Comment by pebiantara** [comment]
closing and merge to https://github.com/SupernovaBrands/sandandsky-styleguides/pull/66

**Pull request closed without merging**